--- libacpi-0.2/Makefile~	2007-07-29 14:09:34.000000000 +0200
+++ libacpi-0.2/Makefile	2007-07-29 17:40:18.000000000 +0200
@@ -30,7 +30,7 @@
 
 libacpi.so: ${OBJ}
 	@echo LD $@
-	@${LD} ${SOFLAGS} -o $@.${SOVERSION} ${OBJ}
+	@${LD} ${SOFLAGS} -o ${SONAME} ${OBJ}
 
 test-libacpi: ${OBJ_test}
 	@echo LD $@
@@ -42,13 +42,13 @@
 	@mkdir -p ${DESTDIR}${PREFIX}/include
 	@cp -f libacpi.h ${DESTDIR}${PREFIX}/include
 	@chmod 644 ${DESTDIR}${PREFIX}/include/libacpi.h
-	@echo installing library to ${DESTDIR}${PREFIX}/lib
-	@mkdir -p ${DESTDIR}${PREFIX}/lib
-	@cp -f libacpi.a ${DESTDIR}${PREFIX}/lib
-	@chmod 644 ${DESTDIR}${PREFIX}/lib/libacpi.a
-	@cp -f ${SONAME} ${DESTDIR}${PREFIX}/lib/
-	@chmod 644 ${DESTDIR}${PREFIX}/lib/${SONAME}
-	@ln -s ${SONAME} ${DESTDIR}${PREFIX}/lib/libacpi.so
+	@echo installing library to ${DESTDIR}${LIBDIR}
+	@mkdir -p ${DESTDIR}${LIBDIR}
+	@cp -f libacpi.a ${DESTDIR}${LIBDIR}
+	@chmod 644 ${DESTDIR}${LIBDIR}/libacpi.a
+	@cp -f ${SONAME} ${DESTDIR}${LIBDIR}/
+	@chmod 755 ${DESTDIR}${LIBDIR}/${SONAME}
+	@ln -s ${SONAME} ${DESTDIR}${LIBDIR}/libacpi.so
 	@echo installing test-libacpi to ${DESTDIR}${PREFIX}/bin
 	@mkdir -p ${DESTDIR}${PREFIX}/bin
 	@cp -f test-libacpi ${DESTDIR}${PREFIX}/bin
@@ -57,18 +57,16 @@
 	@mkdir -p ${DESTDIR}${MANPREFIX}/man3
 	@cp libacpi.3 ${DESTDIR}${MANPREFIX}/man3
 	@echo installing documentation and misc files to ${DESTDIR}${PREFIX}/share/doc/libacpi
-	@mkdir -p ${DESTDIR}${PREFIX}/share/doc/libacpi
-	@cp -r AUTHORS CHANGES README LICENSE doc ${DESTDIR}${PREFIX}/share/doc/libacpi
 	@echo finished installation
 
 uninstall:
 	@echo removing header file from ${DESTDIR}${PREFIX}/include
 	@rm -f ${DESTDIR}${PREFIX}/include/libacpi.h
-	@echo removing library file from ${DESTDIR}${PREFIX}/lib
+	@echo removing library file from ${DESTDIR}${LIBDIR}
 	@rm -f ${DESTDIR}${PREFIX}/lib/libacpi.a
-	@echo removing shared object file from ${DESTDIR}${PREFIX}/lib
-	@rm -f ${DESTDIR}${PREFIX}/lib/libacpi.so
-	@rm -f ${DESTDIR}${PREFIX}/lib/${SONAME}
+	@echo removing shared object file from ${DESTDIR}${LIBDIR}
+	@rm -f ${DESTDIR}${LIBDIR}/libacpi.so
+	@rm -f ${DESTDIR}${LIBDIR}/libacpi.so.${SONAME}
 	@echo removing test-libacpi client from ${DESTDIR}${PREFIX}/bin
 	@rm -f ${DESTDIR}${PREFIX}/bin/test-libacpi
 	@echo removing manual page from ${DESTDIR}${MANPREFIX}/man3
--- libacpi-0.2/config.mk~	2007-07-29 14:09:34.000000000 +0200
+++ libacpi-0.2/config.mk	2007-07-29 17:46:40.000000000 +0200
@@ -5,6 +5,7 @@
 # customize below to fit your system
 PREFIX = /usr/local
 MANPREFIX = ${PREFIX}/share/man
+LIBDIR = ${PREFIX}/lib
 
 # flags
 SOFLAGS = -shared -Wl,-soname,${SONAME}
